packages/fxa-settings/src/components/DataBlock/index.tsx

```diff
@@ -19,12 +19,16 @@ type actionFn = (action: actions) => void;
 export type DataBlockProps = {
   value: string | string[];
   prefixDataTestId?: string;
+  separator?:string,
+  onCopy?: (event:any) => void,
```

Comment:
I like to try and avoid `any` where possible; `React.ClipboardEvent<HTMLDivElement>` should work here. You'll also need to update the function to `copyRecoveryCodes(event: React.ClipboardEvent<HTMLDivElement>)`

Reply:
Good call!